test/fuzz/transformation_add_function_test.cpp

```diff
@@ -58,6 +58,56 @@ std::vector<protobufs::Instruction> GetInstructionsForFunction(
   return result;
 }
 
+// Returns true if and only if every pointer parameter and variable associated
+// with |function_id| in |context| is known by |fact_manager| to be arbitrary,
+// with the exception of |loop_limiter_id|, which must not be arbitrary.  (It
+// can be 0 if no loop limiter is expected, and 0 should not be deemed
+// arbitrary).
+bool AllVariablesAndParametersExceptLoopLimiterAreArbitrary(
+    opt::IRContext* context, const FactManager& fact_manager,
+    uint32_t function_id, uint32_t loop_limiter_id) {
```

Comment:
Could mention that you don't have tests with multiple loops? Hence you only need one loop_limiter_id

Reply:
There is actually only one loop limiter per function.